modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/trash/KBArticleTrashHandler.java

```diff
@@ -103,6 +106,17 @@ public TrashedModel getTrashedModel(long classPK) {
 			classPK, WorkflowConstants.STATUS_ANY);
 	}
 
+	@Override
+	public TrashRenderer getTrashRenderer(long classPK) throws PortalException {
```

Comment:
Question:

why there is no need of getTrashRenderer on `KBFolderTrashHandler`?

Reply:
KBFolders assets are not rendered